python/tvm/contrib/emcc.py

```diff
@@ -44,25 +42,25 @@ def create_js(output,
         The compile string.
     """
     cmd = [cc]
-    cmd += ["-Oz"]
-    if not side_module:
-        cmd += ["-s", "RESERVED_FUNCTION_POINTERS=2"]
-        cmd += ["-s", "NO_EXIT_RUNTIME=1"]
-        extra_methods = ['cwrap', 'getValue', 'setValue', 'addFunction']
-        cfg = "[" + (','.join("\'%s\'" % x for x in extra_methods)) + "]"
-        cmd += ["-s", "EXTRA_EXPORTED_RUNTIME_METHODS=" + cfg]
-    else:
-        cmd += ["-s", "SIDE_MODULE=1"]
-    cmd += ["-o", output]
+    cmd += ["-O3"]
+
+    cmd += ["-std=c++14"]
+    cmd += ["-s", "ERROR_ON_UNDEFINED_SYMBOLS=0"]
+    cmd += ["-s", "STANDALONE_WASM=1"]
+
```

Comment:
By default, emcc generates a wasm file whose max memory size is 16 MB.  It is too small to use for deep learning.  We should extend the size by adding "-s TOTAL_MEMORY=[mem_size]" at least.  Another way is adding "-s ALLOW_MEMORY_GROWTH=1" and calling memory.grow() when necessary.  It looks hard to implement, but we can avoid hard-coding the maximum memory size.

Reply:
We can certainly enable memory growth